tests/test_tem_inputs.py

```diff
@@ -204,6 +216,7 @@ def test_fill_parameters_dictionary_min():
     detector_q_efficiency = 0.1
     mtf_params = [0.1, 0.0, 0.7, 0, 0]
     log_file = "itslog.log"
+    n_samples = 5
```

Comment:
have n_particles as input. From particle size and size of micrograph, figure out how many micrographs needs to be simulated, then give run this many micrographs.